src/en/spyfakku/src/eu/kanade/tachiyomi/extension/en/spyfakku/SpyFakku.kt

```diff
@@ -83,78 +80,102 @@ class SpyFakku : HttpSource() {
                         if (filter.state.isNotEmpty()) {
                             terms += filter.state.split(",").filter { it.isNotBlank() }.map { tag ->
                                 val trimmed = tag.trim().replace(" ", "_")
-                                (if (trimmed.startsWith("-")) "-" else "") + filter.type + "&:" + trimmed.removePrefix("-")
+                                (if (trimmed.startsWith("-")) "-" else "") + filter.type + ":" + trimmed.removePrefix("-")
                             }
                         }
                     }
 
                     else -> {}
                 }
             }
-            addPathSegment("search")
             addQueryParameter("q", terms.joinToString(" "))
             addQueryParameter("page", page.toString())
         }.build()
         return GET(url, headers)
     }
 
     override fun mangaDetailsRequest(manga: SManga): Request {
-        return GET(baseUrl + manga.url + ".json", headers)
+        return GET(baseApiUrl + manga.url, headers)
     }
 
     override fun pageListRequest(chapter: SChapter): Request {
-        return GET(baseUrl + chapter.url + ".json", headers)
+        return GET(baseApiUrl + chapter.url, headers)
     }
 
     override fun getFilterList() = getFilters()
 
-    private val dateFormat = SimpleDateFormat("EEEE, d MMM yyyy HH:mm (z)", Locale.ENGLISH)
+    private val dateReformat = SimpleDateFormat("EEEE, d MMM yyyy HH:mm (z)", Locale.ENGLISH)
+    private val releasedAtFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.ENGLISH).apply {
+        timeZone = TimeZone.getTimeZone("UTC")
+    }
+    private val createdAtFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.ENGLISH).apply {
+        timeZone = TimeZone.getTimeZone("UTC")
+    }
+
     private fun Hentai.toSManga() = SManga.create().apply {
         title = this@toSManga.title
-        url = "/archive/$id/$slug"
-        author = artists?.joinToString { it.value }
+        url = "/g/$id"
```

Comment:
will the old urls still work? if not then add some logic to handle them in `mangaDetailsRequest` and `chapterListRequest`